### PR TITLE
extends index

### PR DESCRIPTION
**Pull request by geniusgordon**

### CODE REVIEW

File: problem/templates/problem/detail.html [hunk 1]
```diff
@@ -21,8 +21,8 @@
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE.
 -->
-{% extends 'problem/nav.html' %} {% block Body %}
-<div id="Body">
-  {% include 'problem/problem.html' %}
+{% extends 'problem/base.html' %} {% block body_block %}
+<div class="container" id="Body">
+    {% include 'problem/problem.html' %}
```
Comment (nickelcheng) [inline problem/templates/problem/detail.html]: Please take care of the indent.

**Comment by nickelcheng** [comment]
This PR is ok for me. If @bruce3557  doesn't have other suggestions, we can merge it.

File: problem/static/problem/css/problem.css [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#Body {
```
Comment (nickelcheng) [inline problem/static/problem/css/problem.css]: Please add license.

**Comment by nickelcheng** [comment]
This PR is ok for me after the license added. Does @bruce3557 have other suggestions?

**Comment by bruce3557** [comment]
If no other naming rule suggestion, I pass this PR.

**Comment by nickelcheng** [comment]
Come to merge.

**Pull request merged**